packages/firebase_messaging/firebase_messaging/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 13.0.0
+
+> Note: This release has breaking changes.
+
+ - **BREAKING** **FEAT**: android 13 notifications permission request ([#9348](https://github.com/firebase/flutterfire/issues/9348)). ([43b3b06b](https://github.com/firebase/flutterfire/commit/43b3b06b64739658f79c994110654f5a56abca05))
```

Comment:
I think we should mention why is this breaking with more information like we mentioned here: https://github.com/firebase/flutterfire/pull/9348/files#r946825531

Reply:
Ah, I added that to the description when I squashed it which I thought added to the Changelog 🤔 . Will update 👍

Reply:
Huuum do we have the right GitHub setting 🤔

Reply:
Either that or melos 🤷 . Not sure without invesitgating.

Reply:
> Ah, I added that to the description when I squashed it which I thought added to the Changelog 🤔 . Will update 👍

Melos only adds it if you do it in the following format in the commit body (conventional commits format):

```

BREAKING: Some text here about why it's breaking.

```